src/Tribe/Updater.php

```diff
@@ -198,10 +185,21 @@ public function flush_rewrites() {
 		add_action( 'wp_loaded', 'flush_rewrite_rules' );
 	}
 
+	/**
+	 * Set the Capabilities for Events and Related Post Types.
+	 *
+	 * @since TBD - change method of calling set_capabilities.
+	 */
 	public function set_capabilities() {
-		$this->capabilities = new Tribe__Events__Capabilities();
-		add_action( 'wp_loaded', array( $this->capabilities, 'set_initial_caps' ) );
-		add_action( 'wp_loaded', array( $this, 'reload_current_user' ), 11, 0 );
+		/* @var Tribe__Events__Capabilities $capabilities */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 229ce064aeba0ba487fab6fb0c7883c8</sub>